### DISCUSSION TITLE
Unable to cut segments

### DISCUSSION THREAD

**Discussion by frantathefranta** [Hypfer/Valetudo#1966] [Q&A / Support] [answered]
Hi, I'm enjoying Valetudo a lot with my Dreame L10 Pro, but there is one thing I can't figure out. Any way I try it, it's impossible to split segments. I get this error when I press the **Split Room** button:

`An error occurred while sending command to MapSegmentEditCapability: Failed to split segment. Both ends of the cutting line need to be connected with a wall surrounding the chosen segment.`

This is an example of where I'm putting this splitting line:
<img width="602" alt="Screenshot 2024-01-17 at 8 13 29 PM" src="https://github.com/Hypfer/Valetudo/assets/64412753/ee8c0a3d-812d-4675-aee2-52cb98949f75">

I tried to align it vertically, horizontally, diagonally, but nothing seems to be successful. 

Thanks for any advice!

**Comment by Hypfer** [comment, marked as answer]
When this happens, try making the line longer to cover the whole segment area.

In some room layouts it might also be required to cut more than you want and then merge the excess back together later.

**Reply by frantathefranta** [reply]
Really not sure what was going on before, but without updating anything, I can now cut segments without any issues.
